### PR TITLE
[Enhancement] Optimize display of explain costs

### PR DESCRIPTION
**Pull request by liuyehcf**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Problem Summary(Required) ：

Add explain information of ExchangeNode and TableFunctionNode

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 22 / 24 (91.67%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/planner/ExchangeNode.java|17|19|89.47%|[213, 214]|
|:large_blue_circle:|com/starrocks/planner/TableFunctionNode.java|5|5|100.00%|[]|

File: fe/fe-core/src/main/java/com/starrocks/planner/ExchangeNode.java [hunk 1]
```diff
@@ -202,11 +202,31 @@ protected String debugString() {
...
+        if (detailLevel == TExplainLevel.VERBOSE) {
+            if (distributionType != null) {
+                output.append(detailPrefix).append("distribution type: ")
+                        .append(distributionType).append('\n');
+            }
+            if (partitionType != null) {
+                output.append(detailPrefix).append("partition type: ")
+                        .append(partitionType).append('\n');
+            }
+            if (partitionExprs != null && !partitionExprs.isEmpty()) {
```
Comment (packy92) [inline fe/fe-core/src/main/java/com/starrocks/planner/ExchangeNode.java:216]: can replace it with `CollectionUtils.isNotEmpty(partitionExprs)`

**Review by murphyatwork — DISMISSED** [review]

**Comment by mofeiatwork** [comment]
run starrocks_admit_test

**Comment by liuyehcf** [comment]
run starrocks_admit_test

**Comment by liuyehcf** [comment]
run starrocks_admit_test

**Comment by liuyehcf** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 22 / 24 (91.67%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/planner/ExchangeNode.java|17|19|89.47%|[213, 214]|
|:large_blue_circle:|com/starrocks/planner/TableFunctionNode.java|5|5|100.00%|[]|

**Review by satanson — APPROVED** [review]

**Review by packy92 — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/planner/ExchangeNode.java (hunk 1 above)
Comment (liuyehcf) [reply fe/fe-core/src/main/java/com/starrocks/planner/ExchangeNode.java]: updated

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by liuyehcf** [comment]
run starrocks_admit_test

**Review by kangkaisen — APPROVED** [review]

**Pull request merged**